lib/preload/redir-stdout.js

```diff
@@ -1,6 +1,11 @@
 'use strict'
 const net = require('net')
 
+try {
+  // Skip redirecting stdout in Worker threads.
+  if (!require('worker_threads').isMainThread) return;
```

Comment:
Lint is failing. Would you mind adding an exception?

Reply:
@mcollina Ugh, just seeing that … since this is a parsing error for the linter, not a linter error, I’m not sure there’s a way to add an exception…?

I’ve worked around this the awkward way, wrapping the rest of the file in `if (!isWorker)`. I hope that’s okay?